### PR TITLE
airwave: fix VST SDK and Qt plugins

### PR DESCRIPTION
**Pull request by egasimus**
###### Motivation for this change
* VST SDK 3.6.8 (`vstsdk368_08_11_2017_build_121.zip`) has fallen off the Internet.
* VST SDK 3.6.14 (`vst-sdk_3.6.14_build-24_2019-11-29.zip`) can be automatically downloaded from Steinberg's servers or from GitHub but does not contain the required VST2 SDK headers.

###### Things done

* Made the package use VST SDK 3.6.6 (`vstsdk366_27_06_2016_build_61`).
* Raised an [issue on the Steinberg VST SDK GitHub](https://github.com/steinbergmedia/vst3sdk/issues/51).
* Also added `qt5.wrapQtAppsHook` into nativeBuildInputs as per #65399 (`qt5.mkDerivation` is not an option because of `multiStdenv.mkDerivation` - but hey, now it builds and runs!)

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by worldofpeace** [comment]
> Also added qt5.wrapQtAppsHook into nativeBuildInputs as per #65399 (qt5.mkDerivation is not an option because of multiStdenv.mkDerivation - but hey, now it builds and runs!)

You can do `mkDerivationWith multiStdenv.mkDerivation`

File: pkgs/applications/audio/airwave/default.nix [hunk 1]
```diff
@@ -42,7 +41,7 @@ multiStdenv.mkDerivation {
 
   src = airwave-src;
 
-  nativeBuildInputs = [ cmake makeWrapper ];
+  nativeBuildInputs = [ cmake makeWrapper qt5.wrapQtAppsHook ];
 
   buildInputs = [ file libX11 qt5.qtbase wine-xembed ];
```
Comment (worldofpeace) [inline pkgs/applications/audio/airwave/default.nix]: we should never reference `qt5` directly, and instead should use the `libsForQt5.callPackage` function to bring all attributes within `qt5` into scope.

Comment (worldofpeace) [reply pkgs/applications/audio/airwave/default.nix]: see https://nixos.org/nixpkgs/manual/#sec-language-qt

**Comment by SuperSandro2000** [comment]
Closing due to inactivity. Feel free to reopen the discussion.

**Pull request closed without merging**